v0/destinations/mp/transform.js

```diff
@@ -15,9 +15,14 @@ const {
   getSuccessRespEvents,
   CustomError,
   isAppleFamily,
-  getFullName
+  getFullName,
+  extractCustomFields
 } = require("../../util");
-const { ConfigCategory, mappingConfig } = require("./config");
+const {
+  ConfigCategory,
+  mappingConfig,
+  MP_IDENTIFY_EXCLUSION_LIST
```

Comment:
We are adding the address to the exclusion_list, but what if someone sends the address as a string?

Reply:
We are mapping 
`$country_code`, `$city`, `$region` from the `address` field, we don't add the address in the exclusion list it will come up as stringified JSON in the dashboard, which is unwanted.

Reply:
Yeah, that makes sense but if someone sends the address as a string then we are discarding it. It is not wanted.

Reply:
Yes as we have mentioned in our spect address is expected as `object`